File: labs/solidworks/swex/pmpage/controls/combo-box/ComboBox.ItemsText.vb
Imports CodeStack.SwEx.Common.Attributes
Imports CodeStack.SwEx.My.Resources
Imports CodeStack.SwEx.PMPage.Attributes
Imports SolidWorks.Interop.swconst

Public Class ComboBoxDataModel

    Public Enum OptionsCustomized_e
        <Title("First Option")>
        Option1
        <Title(GetType(Resources), NameOf(Resources.Option2Title))>
        Option2
    End Enum

    Public Property Options2 As OptionsCustomized_e

End Class

File: labs/solidworks/swex/pmpage/controls/combo-box/ComboBox.ItemsText.cs
using CodeStack.SwEx.Common.Attributes;
using CodeStack.SwEx.PMPage.Attributes;
using CodeStack.SwEx.Properties;
using SolidWorks.Interop.swconst;

public class ComboBoxDataModel
{
    public enum OptionsCustomized_e
    {
        [Title("First Option")] //static title
        Option1,

        [Title(typeof(Resources), nameof(Resources.Option2Title))] //title loaded from resources
        Option2
    }

    public OptionsCustomized_e Options2 { get; set; }
}
